### PR TITLE
Text updates

### PR DESCRIPTION
**Pull request by michaelhush**

### CODE REVIEW

**Comment by michaelhush** [comment]
@saltysealion @stevegore CircleCI seems to be failing, but I only changed the text of the numerical notebook. Not sure what is going on.

File: examples/optimal_control_optimization.ipynb [hunk 1]
```diff
@@ -6,18 +6,20 @@
...
-    "3. Export the optimized control.\n",
+    "2. Run an optimal-control numeric optimization.\n",
+    "3. Export the optimal-control solution.\n",
     "\n",
-    "In this notebook we present the optimal optimization of:\n",
+    "In this notebook we present optimal-control solutions designed to minimize leakage errors in two operations of varying complexity:\n",
     "\n",
-    "* An Hadamard gate on an individual qubit with one extra leakage level.\n",
-    "* An iSWAP gate on two qubits, each with a leakage level.\n",
+    "* A Hadamard gate perforemed on an individual qubit with one extra leakage level.\n",
```
Comment (stevegore) [inline examples/optimal_control_optimization.ipynb]: Typo :)

**Comment by stevegore** [comment]
@michaelhush The ```development``` branch was already failing.

**Comment by stevegore** [comment]
Comparison to last good build: https://github.com/qctrl/python/compare/915246f4ddd4c68a42ff438f4dd5166001837796...optimal_optimization_update

**Comment by stevegore** [comment]
@stabacco  @saltysealion might be in a better place to help with the failed scenarios.

**Comment by michaelhush** [comment]
@stabacco fixed the file output

**Comment by saltysealion** [comment]
@michaelhush at the time of branching, the development branch was failing builds. It's green now so if you rebase it should be fine.

**Pull request merged**